browser/skus/skus_sdk_mojom_impl.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/skus/skus_sdk_mojom_impl.h"
+
+#include <memory>
+#include <utility>
+
+#include "brave/browser/skus/skus_sdk_service_factory.h"
+#include "brave/components/skus/browser/skus_sdk_service.h"
+#include "content/public/browser/browser_context.h"
+
+namespace brave_rewards {
+
+SkusSdkMojomImpl::SkusSdkMojomImpl(content::BrowserContext* context)
```

Comment:
This class should not exist, you should be binding to the service using mojo just like in Brave wallet

Reply:
done